radicle-crypto/src/ssh/keystore.rs

```diff
@@ -3,9 +3,13 @@ use std::path::{Path, PathBuf};
 use std::{fs, io};
 
 use thiserror::Error;
+use zeroize::Zeroizing;
 
 use crate::{KeyPair, PublicKey, SecretKey, Signature, Signer, SignerError};
 
+/// A secret key passphrase.
+pub type Passphrase = Zeroizing<String>;
```

Comment:
Is it possible to `Zeroizing<&str>`? :thinking:

Reply:
Hehe nope, I tried that. I wonder if it's possible in theory though, since it has to have control over the drop, so it might not be possible with borrows.

Reply:
Ah ya, good point :) Slight optimization might be to do:
```
pub struct Passphrase(Zeroizing<String>);

impl From<&str> for Zeroizing {
  /* yada yada */
}

impl From<String> for Zeroizing {
  /* yada yada */
}
```

Would save some `to_owned().into()`s.

Reply:
Yeah good idea :thinking: